drivers/i3c/i3c_common.c

```diff
@@ -97,6 +97,7 @@ int i3c_addr_slots_init(const struct device *dev)
 	sys_slist_init(&data->attached_dev.devices.i3c);
 	sys_slist_init(&data->attached_dev.devices.i2c);
 
+	/* Address restrictions (ref 5.1.2.2.5, Specification for I3C v1.0) */
```

Comment:
nit: zephyr implements the v1.1.1 specification here, the section number is the same

Reply:
adjusted